copper_sdk/copper.py

```diff
@@ -72,6 +72,7 @@ def api_call(self, method, endpoint, opts = None):
               print(optsJson)
 
         # dynamically call method to handle status change
+        self.retry_count = 0
```

Comment:
Can we add the unit test to cover this case?

Reply:
https://github.com/lendsmartlabs/worklets/pull/1370
ADDED